client/tailwind.config.js

```diff
@@ -7,22 +7,44 @@ module.exports = {
     },
     extend: {
       colors: {
-        bcBluePrimary: '#003366',
         bcYellowPrimary: '#FCBA19',
         bcBlack: '#313132',
+        bcDeepBlack: '#272833',
```

Comment:
I'm curious about these extra colors, the BC design system is pretty limited as far as I'm aware, where do these come from?

https://developer.gov.bc.ca/Design-System/Colour-Palette

Reply:
Hey Arran, good question! I should have clarified, I copied them from our other BC gov project (Team Based Care Mapping) which had all these approved (they had to use some more, so I grabbed them just in case).